Taskfile.yml

```diff
@@ -943,7 +979,17 @@ tasks:
       # This below is added to test if ASO works fine in namespaces other than 'azureserviceoperator-system"
       NAMESPACE: "custom-namespace"
     cmds:
-      - task: controller:kind-create
+      - task: controller:kind-create-wi
+      - task: controller:create-mi-for-workload-identity
+        vars:
+          # Use the tenant1-system namespace as that's where the single tenant operator will be installed
+          SUBJECT: "system:serviceaccount:tenant1-system:azureserviceoperator-default"
+          # We have to set RESOURCE_GROUP and ISSUER here because if we don't, the cached variables
+          # will be used from the create-mi-for-workload-identity task in ci. See: https://github.com/go-task/task/issues/524
+          RESOURCE_GROUP:
+            sh: "cat ./{{.KIND_WORKLOAD_IDENTITY_PATH}}/azure/rg.txt #controller:kind-create-multitenant-cluster-helm"
+          ISSUER:
+            sh: "cat ./{{.KIND_WORKLOAD_IDENTITY_PATH}}/azure/saissuer.txt #controller:kind-create-multitenant-cluster-helm"
```

Comment:
I think you can use `run: always` on the task to avoid this workaround.

Reply:
or we run the CI tasks with `--force` always?

Reply:
The issue isn't that the task doesn't run. The task runs. See https://github.com/go-task/task/issues/524 (linked in the above comment) for why this is needed.